apps/crn-server/src/handlers/webhooks/cronjob-sync-orcid.ts

```diff
@@ -56,3 +57,7 @@ export const handler = async (): Promise<lambda.Response> => {
     statusCode: 200,
   };
 };
+
+export const unloggedHandler = rawHandler;
```

Comment:
I really don't understand why we are exposing the raw handler over just the sentry wrapped one. It is whats used in production so it improves the integration tests quality. We can omit DSN when we run the tests so nothing is logged into sentry like we do for FE.

Reply:
By doing this if someone changes the sentry wrapper in a way that doesn't work for event handlers it will be caught automatically which seems important for something that is hard to test locally

Reply:
Our handlers are inconsistent, some take just `event`, and some take `event, context`, and sometimes these shapes are custom formats or pulled in from different libraries. 

In order to make all of the tests work against the new handlers, we would need to rewrite all of the handler tests and create proper `context` fixtures for each one. I spoke to @peterstarling about doing this, and we agreed it needs to be done but it's out of scope for this ticket.

We agreed that we'd expose the old handlers so that the tests can continue to run and pass code coverage, knowing that we're testing everything except the wrapper, and build new tests for the wrapper to ensure that it calls the correct functions (we don't want to test the underlying `Sentry.AWSLambda` module.

I agree with you that we should get rid of these but only after we've:

- Created a proper `context` fixture;
- Properly rewritten all of our handlers to take AWS `event` and `context`;
- Rewritten all of the tests to provide `event` and `context`.

When we do that, we can mock the calls to `Sentry` to do nothing in the tests, or provide no DSN.